app/services/listing_index_service/search/zappy_adapter.rb

```diff
@@ -54,6 +54,8 @@ def search(community_id:, search:, includes: nil)
     def format_params(original)
       {
        :'search-keywords' => original[:keywords],
+       :'search-latitude' => original[:lat],
+       :'search-longitude' => original[:lng],
```

Comment:
Probably something shorter. 
"lat", "search-lat" or "search-coords" => "28.613939,77.209021" / or something like that.